pkg/sdk/extensions/api/v1alpha1/hosttailer_types.go

```diff
@@ -82,6 +82,8 @@ type FileTailer struct {
 	Disabled bool `json:"disabled,omitempty"`
 	// Set the limit of the buffer size per monitored file
 	BufferMaxSize string `json:"buffer_max_size,omitempty"`
+	// Read_From_Head
+	ReadFromHead string `json:"read_from_head,omitempty"`
```

Comment:
This might be better typed as a `bool` and formatted with `%t`, then you wouldn't have to worry about setting a default value for it.